orchagent/orch.cpp

```diff
@@ -239,23 +247,67 @@ void ConsumerBase::dumpPendingTasks(vector<string> &ts)
 
 void Consumer::execute()
 {
-    // ConsumerBase::execute_impl<swss::ConsumerTableBase>();
     SWSS_LOG_ENTER();
 
-    auto table = static_cast<swss::ConsumerTableBase *>(getSelectable());
-    std::deque<KeyOpFieldsValuesTuple> entries;
-    table->pops(entries);
+    size_t total_size = 0;
+
+    while (true)
```

Comment:
This while loop cause high priority notification blocked by low priority notification bug, and removed by this PR:
https://github.com/sonic-net/sonic-swss/pull/3269

Why add it back?

Reply:
thanks for reminding...have added a commit to remove the loop : )

Reply:
# Closed